pkgs/development/python-modules/cfn-lint/default.nix

```
@@ -60,9 +63,12 @@ buildPythonPackage rec {
     "cfnlint.transform"
   ];
 
+  checkInputs = [ pytestCheckHook mock ];
```

needs to render a template
```suggestion
  checkInputs = [ pytestCheckHook mock pydot ];
```